app/models/harvested_resource.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Model for a single metadata file that was harvested
+class HarvestedResource < ApplicationRecord
+  belongs_to :harvest
+  belongs_to :pipeline
+
+  def content
+    resource_content.body
+  end
+
+  # Return an identifier for use in the IR.  This is a hack that is needed
+  # because some of our metadata files don't have identifiers encodeded internally.
+  # Don't use this method if you have a better way of getting an identifier.
+  def identifier
```

Comment:
@aaron-collier and I were talking about maybe pushing this identifier logic down into the traject configs, and just making sure the original filename gets passed through (where it could then apply this logic). What do you think?